src/pages/contact_us/_affiliates.tsx

```diff
@@ -91,36 +63,15 @@ export const Affiliates = () => {
             <CardContainer>
                 <Item>
                     <Header as="h3" type="section-title">
-                        {localize('Affiliates')}
-                    </Header>
-                    <Text m="0.8rem 0">
-                        {localize('For enquiries about our affiliate and IB programmes')}
-                    </Text>
-                    <LinkText
-                        color="red"
-                        size="var(--text-size-m)"
-                        href="mailto:partners@deriv.com"
-                        external
-                    >
-                        partners@deriv.com
-                    </LinkText>
-                </Item>
-                <Item>
-                    <Header as="h3" type="section-title">
-                        {localize('Business development')}
+                        <Localize translate_text="Business Partnership" />
                     </Header>
-                    <Text m="0.8rem 0">{localize('For collaboration opportunities')}</Text>
-                    <LinkText
-                        color="red"
-                        size="var(--text-size-m)"
-                        href="mailto:marketing@deriv.com"
-                        external
-                    >
-                        marketing@deriv.com
-                    </LinkText>
+                    <TextWrapper>
+                        <Text m="0.8rem 0">
+                            <Localize translate_text="Questions about becoming our affiliates or other business collaboration opportunities? " />
+                        </Text>
+                        <LiveChatLinkText text="Contact us via live chat" />
```

Comment:
```suggestion
        <LiveChatLinkText text="_t_Contact us via live chat_t_" />
```
since you used `text` directly in `LiveChatLinkText`, you have to wrap the text with `_t_`.